tmk_core/common.mk

```diff
@@ -15,6 +15,7 @@ TMK_COMMON_SRC +=	$(COMMON_DIR)/host.c \
 	$(COMMON_DIR)/usb_util.c \
 	$(PLATFORM_COMMON_DIR)/suspend.c \
 	$(PLATFORM_COMMON_DIR)/timer.c \
+	$(PLATFORM_COMMON_DIR)/wait.c \
```

Comment:
Also needs to be added for`arm_atsam` if you do this, otherwise...
```
 | arm-none-eabi-gcc: error: .build/obj_massdrop_ctrl_default/common/arm_atsam/wait.o: No such file or directory
```

Reply:
Now fixed